src/main/java/uk/ac/sanger/sccp/stan/GraphQLMutation.java

```diff
@@ -556,7 +568,7 @@ public DataFetcher<Work> updateWorkPriority() {
 
     public DataFetcher<Work> updateWorkOmeroProject() {
         return dfe -> {
-            User user = checkUser(dfe, User.Role.normal);
+            User user = checkUser(dfe, User.Role.enduser);
```

Comment:
I made the change to allow endusers the privilege to update omero project and dnap study for works that they created because it seemed to make sense in the feature. If it's not needed, then it's fine.

Reply:
Ok, the ability to update their own work make sense 👍